index.html

```diff
@@ -156,9 +179,11 @@ <h3 class="text-light ff-inai">
           <li class="text-light mg-1"><a href="#"> Terminal </a></li>
         </ul>
       </div>
+    </div>
     </section>
-    <footer class="container"> 
-      <div class="contact-form mg-8 ">
+
+    <footer> 
+      <div class="contact-form container mg-8 flex">
```

Comment:
- [ ] Kindly increase the width of the contact form and make the `Get in touch` button align the inputs from the beginning(not in the center) to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/165759287-285a1255-3745-4905-bcb4-a71cedf4f3d5.png) | ![image](https://user-images.githubusercontent.com/24703171/165759070-907e9b97-b85d-4103-9477-7a7faee1de32.png) |